### PR TITLE
Various fixes to assignees and due date drop downs

### PR DESCRIPTION
**Pull request by DavidHollins6**
### 💬 Description
Comments on the code!

### ✅ Checklist
- [ ] Tests written
- [ ] Browser tested
- [ ] Added to documentation
- [ ] Added to storybook

### CODE REVIEW

File: lib/src/modules/calendar/calendar.scss [hunk 1]
```diff
@@ -109,6 +109,11 @@
     &:hover {
       @apply text-white;
     }
+    &.DayPicker-Day--outside {
+      &:after {
+        @apply bg-transparent;
+      }
+    }
```
Comment (DavidHollins6) [inline lib/src/modules/calendar/calendar.scss:116]: We don't want to put a bg on any days that are `outside` of the current month

File: lib/src/prefabs/assigneeDropdown/AssigneeDropdown.js [hunk 2]
```diff
@@ -70,7 +70,10 @@ export function AssigneeDropdown({
                 <DropdownContent.Footer className="flex">
                   <ButtonSecondaryDanger
                     size={ButtonSecondaryDanger.sizes.sm}
-                    onClick={handleUnassignWithLoader}
+                    onClick={async e => {
+                      await handleUnassignWithLoader(e);
+                      setShowContent(false);
```
Comment (DavidHollins6) [inline lib/src/prefabs/assigneeDropdown/AssigneeDropdown.js:75]: We now close the dropdown after the call has been handled.

File: lib/src/prefabs/dueDateDropdown/DueDateDropdown.js [hunk 3]
```diff
@@ -24,7 +24,12 @@ export function DueDateDropdown({
   disableRemoveButton
 }) {
   return (
-    <Dropdown id={id} autoPosition={autoPosition} onToggle={onToggle}>
+    <Dropdown
+      id={id}
+      autoPosition={autoPosition}
+      onToggle={onToggle}
+      className=" z-20"
```
Comment (DavidHollins6) [inline lib/src/prefabs/dueDateDropdown/DueDateDropdown.js:31]: More `z-index` fun!

**Comment by DavidHollins6** [comment]
[Workflow / assignees sidebar Mob Test](https://trello.com/c/2Wkpes7z/1624-workflow-assignees-sidebar-mob-test)

**Review by AmeeMorris — APPROVED** [review]

**Pull request merged**